### DISCUSSION TITLE
[iOS] PR Review: Comment on Multiline not possible

### DISCUSSION THREAD

**Discussion by malteschlueter** [community/community#72157] [Mobile] [answered]
iOS app version: 1.136.0

If I review a PR I'm able to comment a single line of code by touching the line. 

But if I try to comment a multiline select it's not possible because when I select multiple lines of code (hold touch) the lines are marked and the grey button to submit the review changes to a blue button to comment. But if I touch the button it opens the review submit form instead of the comment form.![image](https://github.com/community/community/assets/13586874/eeeaf3da-4356-4983-a4b2-9f8370df562d)![image](https://github.com/community/community/assets/13586874/668690b1-5b5d-4346-a49a-6aac42c04fb1)

**Comment by aconsuegra** [comment]
Thanks for the report @malteschlueter. I can reproduce the issue, so we'll try to get a fix ready as soon as possible.

**Reply by aconsuegra** [reply, marked as answer]
@malteschlueter We just published a new version in the App Store with a fix. Thanks again for reporting the issue, and I'm sorry if it has caused any inconveniences.

**Reply by malteschlueter** [reply]
No problem, thanks a lot for the fast fix. 
It works now:)
